File: problems/2022_09_22/0242/Solution.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int[] s1 = new int[26];
        int[] s2 = new int[26];

        for (char ch : s.toCharArray()) {
            s1[ch - 'a']++;
        }

        for (char ch : t.toCharArray()) {
            s2[ch - 'a']++;
        }

        return Arrays.equals(s1, s2);
    }
}

File: problems/2022_09_22/0242/Solution.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        sCountMap = self.getCountMap(s)
        tCountMap = self.getCountMap(t)

        return sCountMap == tCountMap

    def getCountMap(self, string):
        countMap = {}
        for i in string:
            if i not in countMap.keys():
                countMap[i] = 1
            else:
                countMap[i] += 1

        return countMap
